### ISSUE TITLE
change location of temporary files

### ISSUE THREAD

**Issue by peeter-t2** [xiaodaigh/disk.frame#253]
Another question. Is there a way to change the temporary file location it uses to build up the chunks?

Namely the command `csv_to_disk.frame` first builds up files in a local temporary folder, before setting up something in the output directory. I would like to customize that folder so that would also be on an external ssd. (I'm working on an ssd, so local memory space is limited.)

Thanks a lot!

**Comment by xiaodaigh** [comment]
Yeah, there is no setting for that. I am mulling over whether to provide such an option.

For now, you need to change the tempdir manually like this https://stackoverflow.com/q/38364431/239923

Sounds like for your use case that is something you should do anyway.

**Comment by peeter-t2** [comment]
Ah, I did not know that this was a variable for R generally. I will then try to change it like this. Thanks a lot!

**Comment by peeter-t2** [comment]
Changing it in .Renviron is proving quite difficult for some reason. File seems to be ok, but RStudio does not seem to be finding/using it. Then I found that unixtools library allows the directory changed directly within R session. Then, even though this directory is changed - csv_to_disk.frame still seems to use the old temporary folder.

Should the following be enough to change this folder for csv_to_disk.frame?

```
#https://www.rforge.net/unixtools/git.html
#install.packages("unixtools",,"http://rforge.net/")
library(unixtools)
set.tempdir("new/temp/folder/")
```

Thanks!

**Comment by peeter-t2** [comment]
Yes, set.tempdir finally seems to work indeed. Not sure what made a difference here, but works now. Thanks!

**Comment by xiaodaigh** [comment]
I wonder if you need to restart

**Comment by peeter-t2** [comment]
Not sure, I tried restarting R console and Rstudio a few times. Did not do a complete computer restart for a while, but I'm not sure it made a difference.
